### DISCUSSION TITLE
More organized skin folders

### DISCUSSION THREAD

**Discussion by Admant3x** [ppy/osu#23158] [Ideas] [answered]
since making skins i've always had trouble organizing my skins or having trouble finding a file. I would like to add the ability to put multiple skin elements in separate folders, for example like **combo numbers** or **hitcirlce numbers** which in the **skin.ini** can be formatted in **fonts/combo/combo.**
I would like to have this feature also for **sounds** or for **scoreentries** that **cannot** be put in separate folders, it would be nice to have this feature as it would be more orderly and easier for skinners to know what to edit or where to find the file, thanks for reading Good day.
![explorer-814x347-04 06AM](https://user-images.githubusercontent.com/121366826/230698591-49a7d624-bb38-440c-ac0e-f8e8b7991804.png)

**Comment by peppy** [comment, marked as answer]
In lazer, we dont' have such a thing as a skin folder. We hope to make everything available and well categorise inside the UI, which should address some of your organisation concerns.
